src/js/node/child_process.ts

```diff
@@ -573,7 +574,7 @@ function spawnSync(file, args, options) {
   });
 
   const result = {
-    signal: null,
+    signal: signalCode,
```

Comment:
Looking the tests and testing locally against node.js we should return null here signalCode is undefined, you can it.if to not run on windows, we already have the test `test/js/node/test/parallel/unhandled-exception-with-worker-inuse.test.js` that will check for the null case

Reply:
good catch it should be null now in the case of no `signalCode` returned, and thanks for the tip didn't know `it.if(condition)` is a thing that's awesome

Reply:
Running the CI again lets see if the tests get something, looks good now.